src/main/resources/hazelcast-config-schema.json

```diff
@@ -1373,20 +1373,535 @@
             "$ref": "#/definitions/AsyncBackupCount"
           },
           "item-listeners": {
-          "type": "array",
-            "items":  {
+            "type": "array",
+            "items": {
               "$ref": "#/definitions/ItemListener"
             }
           },
           "split-brain-protection-ref": {
             "type": "string",
-            "description":  "Adds the Split Brain Protection for this data-structure. You should set this value as a \"split-brain-protection\"'s name."
+            "description": "Adds the Split Brain Protection for this data-structure. You should set this value as a \"split-brain-protection\"'s name."
           },
           "merge-policy": {
             "$ref": "#/definitions/MergePolicy"
           }
         }
       }
+    },
+    "WanEndpointConfig": {
+      "properties": {
+        "name": {
```

Comment:
Is this a required prop?

Reply:
No it's not. See https://github.com/hazelcast/hazelcast/blob/28e882c8050fe901344445f9e24faab4a8801edc/hazelcast/src/main/resources/hazelcast-config-4.1.xsd#L4278